src/components/visualizations/chart/Chart.tsx

```diff
@@ -0,0 +1,115 @@
+// (C) 2007-2018 GoodData Corporation
+import noop = require('lodash/noop');
+import isEqual = require('lodash/isEqual');
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { initChartPlugins } from './highcharts/chartPlugins';
+
+// Have only one entrypoint to highcharts and drill module
+// tslint:disable-next-line
+export const Highcharts = require('highcharts/highcharts');
+// tslint:disable-next-line
+const drillmodule = require('highcharts/modules/drilldown');
+drillmodule(Highcharts);
+initChartPlugins(Highcharts);
+
+export interface ILegendConfig {
+    enabled?: boolean;
+    position?: 'top' | 'left' | 'right' | 'bottom';
+    responsive?: boolean;
+}
+
+export interface IChartLimits {
+    series?: number;
+    categories?: number;
+}
+
+export interface IChartConfig {
+    colors?: string[];
+    type?: string;
+    legend?: ILegendConfig;
+    limits?: IChartLimits;
+    stacking?: boolean;
+}
+
+export interface IChartProps {
+    config: IChartConfig;
+    domProps: any;
+    callback(): void;
+}
+
+export default class Chart extends React.Component<IChartProps, null> {
+    public static propTypes = {
```

Comment:
I'd say we can get rid of React prop types in favor of TS prop types as there's nothing that would be in touch with these components outside of TS world